demos/interactive_face_detection_demo/detectors.cpp

```diff
@@ -213,7 +225,50 @@ void FaceDetection::fetchResults() {
     resultsFetched = true;
     const float *detections = request->GetBlob(output)->buffer().as<float *>();
 
-    for (int i = 0; i < maxProposalCount; i++) {
+    for (int i = 0; i < maxProposalCount && objectSize == 5; i++) {
+        Result r;
+        r.label = 1;
```

Comment:
Should the value be extracted from `labels`?

Reply:
I have an assumption. These are single class detectors so the label is always 1. And dldt forced the model to have that `labels` output even though it always predicts 1. Than if it is possible, you should assure that `num_classes` is 1 (like the one extracted in `if (outputInfo.size() == 1)` with `num_classes = detOutput->get_attrs().num_classes;`). I also suggest clarifying that models' feature in models' descriptions. (Assuming, if it was possible to remove that output from IR, it would have been done.)

Reply:
Changed the code to get label from model output. 

Our current understanding about how output of a alternative (to DetectionOutput) detection model should look like is  to have at least `labels` and `boxes` and despite the fact that current models are single class I would still keep label extraction here.